Sources not building on tns 6.1.2

Hi,

I wanted to build the sources locally, and followed the instructions from README: cloned repo, and then run
```
cd src && npm run setup
```

I am getting this output:
```
checkbox.ios.ts:61:48 - error TS2304: Cannot find name 'BEMBoxType'.

61 const boxTypeProperty = new Property<CheckBox, BEMBoxType>({
                                                  ~~~~~~~~~~

checkbox.ios.ts:65:9 - error TS2304: Cannot find name 'BEMBoxType'.

65       ? BEMBoxType.Circle
           ~~~~~~~~~~

checkbox.ios.ts:66:9 - error TS2304: Cannot find name 'BEMBoxType'.

66       : BEMBoxType.Square;
           ~~~~~~~~~~

checkbox.ios.ts:79:25 - error TS2304: Cannot find name 'BEMCheckBox'.

79   private _iosCheckbox: BEMCheckBox;
                           ~~~~~~~~~~~

checkbox.ios.ts:93:25 - error TS2304: Cannot find name 'BEMCheckBox'.

93     this._iosCheckbox = BEMCheckBox.alloc().initWithFrame(
                           ~~~~~~~~~~~

checkbox.ios.ts:95:10 - error TS2304: Cannot find name 'BEMCheckBox'.

95     ) as BEMCheckBox;
            ~~~~~~~~~~~

checkbox.ios.ts:238:41 - error TS2304: Cannot find name 'BEMBoxType'.

238     this.boxType = this.boxType === 0 ? BEMBoxType.Circle : BEMBoxType.Square;
                                            ~~~~~~~~~~

checkbox.ios.ts:238:61 - error TS2304: Cannot find name 'BEMBoxType'.

238     this.boxType = this.boxType === 0 ? BEMBoxType.Circle : BEMBoxType.Square;
                                                                ~~~~~~~~~~

checkbox.ios.ts:263:16 - error TS2304: Cannot find name 'BEMAnimationType'.

263         return BEMAnimationType.Stroke;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:265:16 - error TS2304: Cannot find name 'BEMAnimationType'.

265         return BEMAnimationType.Fill;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:267:16 - error TS2304: Cannot find name 'BEMAnimationType'.

267         return BEMAnimationType.Bounce;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:269:16 - error TS2304: Cannot find name 'BEMAnimationType'.

269         return BEMAnimationType.Flat;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:271:16 - error TS2304: Cannot find name 'BEMAnimationType'.

271         return BEMAnimationType.Stroke;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:273:16 - error TS2304: Cannot find name 'BEMAnimationType'.

273         return BEMAnimationType.Fade;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:275:16 - error TS2304: Cannot find name 'BEMAnimationType'.

275         return BEMAnimationType.Stroke;
                   ~~~~~~~~~~~~~~~~

checkbox.ios.ts:287:59 - error TS2304: Cannot find name 'BEMCheckBoxDelegate'.

287 class BEMCheckBoxDelegateImpl extends NSObject implements BEMCheckBoxDelegate {
                                                              ~~~~~~~~~~~~~~~~~~~

checkbox.ios.ts:288:34 - error TS2304: Cannot find name 'BEMCheckBoxDelegate'.

288   public static ObjCProtocols = [BEMCheckBoxDelegate];
                                     ~~~~~~~~~~~~~~~~~~~

checkbox.ios.ts:303:48 - error TS2304: Cannot find name 'BEMCheckBox'.

303   public animationDidStopForCheckBox(checkBox: BEMCheckBox): void {
                                                   ~~~~~~~~~~~

checkbox.ios.ts:307:35 - error TS2304: Cannot find name 'BEMCheckBox'.

307   public didTapCheckBox(checkBox: BEMCheckBox): void {
```

And build fails.

I am using nativescript **6.1.2**, running on macOS.

--- a/.gitignore
+++ b/.gitignore
@@ -30,3 +30,4 @@ src/angular/index.metadata.json
 !index.d.ts
 !src/*.d.ts
 !references.d.ts
+!src/typings/*

--- a/src/typings/BEMCheckBox.d.ts
+++ b/src/typings/BEMCheckBox.d.ts
@@ -1,0 +1,191 @@
+
+declare class BEMAnimationManager extends NSObject {
+
+	static alloc(): BEMAnimationManager; // inherited from NSObject
+
+	static new(): BEMAnimationManager; // inherited from NSObject
+
+	animationDuration: number;
+
+	constructor(o: { animationDuration: number; });
+
+	fillAnimationWithBouncesAmplitudeReverse(bounces: number, amplitude: number, reverse: boolean): CAKeyframeAnimation;
+
+	initWithAnimationDuration(animationDuration: number): this;
+
+	morphAnimationFromPathToPath(fromPath: UIBezierPath, toPath: UIBezierPath): CABasicAnimation;
+
+	opacityAnimationReverse(reverse: boolean): CABasicAnimation;
+
+	strokeAnimationReverse(reverse: boolean): CABasicAnimation;
+}
+
+declare const enum BEMAnimationType {
+
+	Stroke = 0,
+
+	Fill = 1,
+
+	Bounce = 2,
+
+	Flat = 3,
+
+	OneStroke = 4,
+
+	Fade = 5
+}
+
+declare const enum BEMBoxType {
+
+	Circle = 0,
+
+	Square = 1
+}
+
+declare class BEMCheckBox extends UIControl implements CAAnimationDelegate {
+
+	static alloc(): BEMCheckBox; // inherited from NSObject
+
+	static appearance(): BEMCheckBox; // inherited from UIAppearance
+
+	static appearanceForTraitCollection(trait: UITraitCollection): BEMCheckBox; // inherited from UIAppearance
+
+	static appearanceForTraitCollectionWhenContainedIn(trait: UITraitCollection, ContainerClass: typeof NSObject): BEMCheckBox; // inherited from UIAppearance
+
+	static appearanceForTraitCollectionWhenContainedInInstancesOfClasses(trait: UITraitCollection, containerTypes: NSArray<typeof NSObject> | typeof NSObject[]): BEMCheckBox; // inherited from UIAppearance
+
+	static appearanceWhenContainedIn(ContainerClass: typeof NSObject): BEMCheckBox; // inherited from UIAppearance
+
+	static appearanceWhenContainedInInstancesOfClasses(containerTypes: NSArray<typeof NSObject> | typeof NSObject[]): BEMCheckBox; // inherited from UIAppearance
+
+	static new(): BEMCheckBox; // inherited from NSObject
+
+	animationDuration: number;
+
+	boxType: BEMBoxType;
+
+	cornerRadius: number;
+
+	delegate: BEMCheckBoxDelegate;
+
+	readonly group: BEMCheckBoxGroup;
+
+	hideBox: boolean;
+
+	lineWidth: number;
+
+	minimumTouchSize: CGSize;
+
+	offAnimationType: BEMAnimationType;
+
+	offFillColor: UIColor;
+
+	on: boolean;
+
+	onAnimationType: BEMAnimationType;
+
+	onCheckColor: UIColor;
+
+	onFillColor: UIColor;
+
+	onTintColor: UIColor;
+
+	readonly debugDescription: string; // inherited from NSObjectProtocol
+
+	readonly description: string; // inherited from NSObjectProtocol
+
+	readonly hash: number; // inherited from NSObjectProtocol
+
+	readonly isProxy: boolean; // inherited from NSObjectProtocol
+
+	readonly superclass: typeof NSObject; // inherited from NSObjectProtocol
+
+	readonly  // inherited from NSObjectProtocol
+
+	animationDidStart(anim: CAAnimation): void;
+
+	animationDidStopFinished(anim: CAAnimation, flag: boolean): void;
+
+	class(): typeof NSObject;
+
+	conformsToProtocol(aProtocol: any /* Protocol */): boolean;
+
+	isEqual(object: any): boolean;
+
+	isKindOfClass(aClass: typeof NSObject): boolean;
+
+	isMemberOfClass(aClass: typeof NSObject): boolean;
+
+	performSelector(aSelector: string): any;
+
+	performSelectorWithObject(aSelector: string, object: any): any;
+
+	performSelectorWithObjectWithObject(aSelector: string, object1: any, object2: any): any;
+
+	reload(): void;
+
+	respondsToSelector(aSelector: string): boolean;
+
+	retainCount(): number;
+
+	self(): this;
+
+	setOnAnimated(on: boolean, animated: boolean): void;
+}
+
+interface BEMCheckBoxDelegate extends NSObjectProtocol {
+
+	animationDidStopForCheckBox?(checkBox: BEMCheckBox): void;
+
+	didTapCheckBox?(checkBox: BEMCheckBox): void;
+}
+declare var BEMCheckBoxDelegate: {
+
+	prototype: BEMCheckBoxDelegate;
+};
+
+declare class BEMCheckBoxGroup extends NSObject {
+
+	static alloc(): BEMCheckBoxGroup; // inherited from NSObject
+
+	static groupWithCheckBoxes(checkBoxes: NSArray<BEMCheckBox> | BEMCheckBox[]): BEMCheckBoxGroup;
+
+	static new(): BEMCheckBoxGroup; // inherited from NSObject
+
+	readonly checkBoxes: NSHashTable<any>;
+
+	mustHaveSelection: boolean;
+
+	selectedCheckBox: BEMCheckBox;
+
+	addCheckBoxToGroup(checkBox: BEMCheckBox): void;
+
+	removeCheckBoxFromGroup(checkBox: BEMCheckBox): void;
+}
+
+declare var BEMCheckBoxVersionNumber: number;
+
+declare var BEMCheckBoxVersionString: interop.Reference<number>;
+
+declare class BEMPathManager extends NSObject {
+
+	static alloc(): BEMPathManager; // inherited from NSObject
+
+	static new(): BEMPathManager; // inherited from NSObject
+
+	boxType: BEMBoxType;
+
+	cornerRadius: number;
+
+	lineWidth: number;
+
+	size: number;
+
+	pathForBox(): UIBezierPath;
+
+	pathForCheckMark(): UIBezierPath;
+
+	pathForFlatCheckMark(): UIBezierPath;
+
+	pathForLongCheckMark(): UIBezierPath;
+}

